lib/commands/location.js

```diff
@@ -19,6 +19,12 @@ export default {
    * @this {XCUITestDriver}
    */
   async getGeoLocation() {
+    // Currently we proxy the setGeoLocation to mobile:setSimulatedLocation for iOS 17+.
+    // It would be helpful to address to use "mobile:getSimulatedLocation" for iOS 17+.
+    if (this.isRealDevice() && this.opts.platformVersion && util.compareVersions(this.opts.platformVersion, '>=', '17.0')) {
+      this.log.info(`Location set by mobile:setSimulatedLocation method should get via mobile:getSimulatedLocation method.`);
```

Comment:
We may set a flag if `mobile:setSimulatedLocation` has been called before and only show such message when the flag is true

Reply:
should get via -> should be retrieved via